.github/workflows/cron.yml

```diff
@@ -10,10 +10,21 @@ jobs:
     #    runs-on: ubuntu-20.04
     runs-on: macos-latest
     steps:
+      - name: Maximize build space
```

Comment:
We can't use `maximize-build-space`
, because with this action our action doesn't finish -
https://github.com/aquasecurity/avd-generator/actions/runs/9200181547